### ISSUE TITLE
Add schematic view of the registered seats (not just a list) on the thank you page

### ISSUE THREAD

**Issue by jonasc** [prechelt/anwesende#9]
To make it easier to verify which seats are already checked in and which are not it might be helpful to have a schematic view of the available seats in the rooms (e.g. a table) and mark the ones which are checked in, maybe even give a number of different people checked in for the same seat (if the systems provides it). Something like the following table where I marked checked-in seats with an asterisk.

    +-----+-----+-----+-----+
    |r1s1 |r1s2 |r1s3*|r1s4 |
    +-----+-----+-----+-----+
    |r2s1 |r2s2*|r2s3 |r2s4 |
    +-----+-----+-----+-----+
    |r3s1 |r3s2 |r3s3*|r3s4 |
    +-----+-----+-----+-----+

I know that this view might not resemble the actual room situation if many qr codes are not placed. But it might actually be possible to shape the table cells in width and height such that they represent the room's `row_dist` and `seat_dist`.

I might try to implement this feature myself but I don't know whether that'll happen in the near future. I'll leave it as a suggestion here until then.

**Comment by prechelt** [comment]
Indeed. For some rooms at my university this will deviate _very far_ from the actual layout. For instance in a seminar room with a U-shaped seating arrangement, this display would be very confusing.

Thanks for the idea and suggestion, but I don't think it would be suitable.  
A much simpler version would be helpful though: Starting a new line in the output for each new row that actually appears.

**Comment by jonasc** [comment]
I agree with the concerns with my original idea. However, maybe not showing all seats but only the occupied ones in a table could give more meaningful information with cluttering? For a U-shaped seating arrangement (here availability of all `r*s1`, `r*s5` and `r5s*`) where some of the seats are occupied it could look like this:

![tmp2](https://user-images.githubusercontent.com/2046766/144569424-c70d1833-ecf4-41e0-90c7-d7f19f4ef370.jpg)

As far as I understand the QR codes in the rooms are arranged in a grid like fashion anyway so I currently don't see that this format would deviate (too) much from the situation in the rooms.

**Comment by prechelt** [comment]
Yes, that makes sense and ought to work.
I see two issues that one should think about and try out:
- Sometimes, more than one person is registered in the same seat (possibly massively so). It would be nice if the table showed this if (but only if) it happens
-  In many rooms, the table will not fit on the screen horizontally. With proper horizontal scrolling, this should not be a problem, but one must make sure the resulting HTML behaves as intended.
